src/data/CoreData.tsx

```diff
@@ -116,6 +121,8 @@ export function useCoreData(): {
     addGlottologLanguages(languagesBySchema, glottologImport || [], manualGlottocodeToISO || {});
     addCLDRLanguageSchema(languagesBySchema, cldrAliases || []);
     addIANAVariantLocales(languagesBySchema, locales, ianaVariants);
+    const variantTags = convertToVariantTagData(ianaVariants || []);
+    connectVariantTags(variantTags, languagesBySchema.Inclusive, locales);
```

Comment:
You'll want to use CLDR here. "Inclusive" prefers 3-letter language codes but "CLDR" uses 2-letter language codes and is to how the IANA catalog is set up.
```suggestion
    connectVariantTags(variantTags, languagesBySchema.CLDR, locales);
```